src/app/app.js

```diff
@@ -0,0 +1,36 @@
+angular.module('graffio', [
+  'graffio.signupController',
+  'graffio.loginController',
+  'graffio.clickController',
+  'ui.router'
+])
+.config(function($stateProvider, $urlRouterProvider) {
+  var rootRef = new Firebase('https://dazzling-heat-2465.firebaseio.com/web/uauth');
+  var user = rootRef.getAuth();
+  if (!user) {
+    $urlRouterProvider.otherwise("/login");
```

Comment:
Is the intent t0 explicit setting otherwise or just trying to route to the particular path?